static/js/client.js

```diff
@@ -103,14 +103,39 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     }
   }
 
-  function updateTitle() {
+  //see http://stackoverflow.com/a/9609450
+  var decodeEntities = (function() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['decodeEntities' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769371567/issues/source?bid=2452379&fileBranchId=2457543#l107)